keps/2936-local-queue-defaulting/kep.yaml

```diff
@@ -0,0 +1,32 @@
+title: LocalQueue defaulting
+kep-number: 2936
+authors:
+  - "@yaroslava-serdiuk"
+status: implementable
+creation-date: 2024-11-25
+reviewers:
+  - "@pbundyra"
+  - "@mwielgus"
+  - "@mimowo"
+approvers:
+  - "@mimowo"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v0.9"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v0.9"
+  beta:
+  stable:
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: KueueDefaulting
```

Comment:
```suggestion
  - name: LocalQueueDefaulting
```

Reply:
I was thinking that the feature could be grown later to have a default CQ, so the default LQ could be created by Kueue. That the reason why the feature gate name is generic enough, to not limit the defaulting functionality to LQ only.

Reply:
I see, but (1) the KEP title is LocalQueue defaulting, (2) Kueue is redundant in the name, because this is the name of the project. So I think CQ defaulting would be a separate feature not under this KEP, and could have a dedicated feature gate, like ClusterQueueDefaulting.

Reply:
Updated